lib/rspec/core/notifications.rb

```diff
@@ -232,9 +232,9 @@ class SkippedExampleNotification < ExampleNotification
       #   RSpec's built-in formatters emit.
       def fully_formatted(pending_number, colorizer=::RSpec::Core::Formatters::ConsoleCodes)
         formatted_caller = RSpec.configuration.backtrace_formatter.backtrace_line(example.location)
-        colorizer.wrap("\n  #{pending_number}) #{example.full_description}", :pending) << "\n     " <<
-          Formatters::ExceptionPresenter::PENDING_DETAIL_FORMATTER.call(example, colorizer) <<
-          "\n" << colorizer.wrap("     # #{formatted_caller}\n", :detail)
+        colorizer.wrap("\n  #{pending_number}) #{example.full_description}", :pending) + "\n     " +
+          Formatters::ExceptionPresenter::PENDING_DETAIL_FORMATTER.call(example, colorizer) +
+          "\n" + colorizer.wrap("     # #{formatted_caller}\n", :detail)
```

Comment:
Opted to wrap all the strings in an array and then call `join` - felt a bit neater than making just one string mutable. Happy to change it though.